internal/pkg/infrastructure/redis/event.go

```diff
@@ -12,6 +12,8 @@ import (
 	"time"
 
 	pkgCommon "github.com/edgexfoundry/edgex-go/internal/pkg/common"
+	pgModels "github.com/edgexfoundry/edgex-go/internal/pkg/infrastructure/postgres/models"
```

Comment:
move `internal/pkg/infrastructure/postgres/models` to `internal/pkg/infrastructure/models`, because the DeviceInfo and Reading models could be opened to other database implementation.
Also, it doesn't look right to put the postgres specific model to DBClient interface.